modules/logs/endpoints.py

```diff
@@ -87,15 +87,17 @@ def read_log_as_json(self, log_type, log_id):  # pylint: disable=no-self-use
         array = []
         with open(filename, 'r', encoding='utf-8') as csv_file:
             reader = csv.reader(csv_file)
+            l_time=0
             for row in reader:
                 try:
-                    array.append([
-                        int((datetime.datetime.strptime(
-                            row[0], "%Y-%m-%d %H:%M:%S") -
-                             datetime.datetime(1970, 1, 1)).total_seconds()) *
-                        1000,
-                        float(row[1])
-                    ])
+                    p_time=int((datetime.datetime.strptime(        # get point timestamp
+                        row[0], "%Y-%m-%d %H:%M:%S") - 
+                        datetime.datetime(1970, 1, 1)).total_seconds()) * 1000
+                     if ( abs(float(row[1])) < 110 ) and           # filter invalid points
+                         ( p_time >= l_time ):                     # only plot if more than 60 sec from previus point
```

Comment:
good idea, but IMO should be configurable...

Reply:
You mean the enable/disable or the time interval ( 60 sec )?

Reply:
@lalo-uy I think you could have a parameter `plot_period` to configure that and have the default of 60.

So you replace the line: `l_time = p_time + 60000;`
With: `l_time = p_time + 1000 * cbpi.get_config_parameter("plot_period", 60)`

It that what you meant @jpgimenez?

Reply:
👍 agree... the parameter is better than a hardcoded value